packages/webapp/components/FooterNavBar.tsx

```diff
@@ -40,20 +41,24 @@ export const tabs: Tab[] = [
   {
     path: '/',
     title: 'Home',
-    icon: (active: boolean) => <HomeIcon secondary={active} size="xxlarge" />,
+    icon: (active: boolean) => (
+      <HomeIcon secondary={active} size={IconSize.Medium} />
```

Comment:
Is this intended to be `Medium` than `Large`? Just commenting for visibility.

Reply:
Yes thanks, they where way bigger then the design intended :)